docs/platform/nodebalancer/getting-started-with-nodebalancers-new-manager/index.md

```diff
@@ -0,0 +1,119 @@
+
+It is important to configure the "default" virtual host in your web server to point at something useful. NodeBalancer will pass the Host header from a browser untouched, so virtual hosts will work entirely normally once you are pointing a domain at the NodeBalancer. It is only mentioned here because testing NodeBalancers can demonstrate quirks in a web server's configuration, particularly when browsing by the NodeBalancer's IP address.
+
+ {{< note >}}
+Health checks are transmitted with a Host header (in HTTP/1.0 mode).
+{{< /note >}}
+
+## Putting the NodeBalancer in Charge
+
+Your NodeBalancer is now working and is able to pass traffic to your web application. It is important to note at this point that configuring the NodeBalancer has not impacted your application's normal operations at all -- you can test NodeBalancer without your users ever knowing. The only exception to this would be when adding a private IP address, if necessary.
+
+Once you are satisfied that NodeBalancer is working normally, you can switch your web application's traffic over to it through DNS.
+
+1.  On the NodeBalancer's overview, you can see its IP address. Take note of the IP address, to use in the A record for your domain.
+
+2.  Edit or create an A record for `www.nodebalancerexample.com`, pointing to your NodeBalancer's IP address.
+
+    [![Adding an A Record.](nodebalancers-add-a-name-small.png "Adding an A Record")](nodebalancers-add-a-name.png)
+
+    Also add an AAAA record for the IPv6 address.
+
+Once the DNS changes propagate, traffic will begin flowing through the NodeBalancer. At this point, you will want to wait at least 24 hours for all caches to catch up to the NodeBalancer before proceeding.
+
+# Additional NodeBalancers and Features
+
+On another Linode, make an exact copy of your current web server. The Linode backups service can be instrumental for doing so, as a snapshot can be restored to any other Linode. Once you have another backend ready, simply repeat step four of [Configuring a NodeBalancer](/docs/platform/nodebalancer/getting-started-with-nodebalancers/#configuring-a-nodebalancer) to add it to the NodeBalancer configuration.
+
+[![Adding another backend to the NodeBalancer's configuration.](nodebalancers-backend-nodes2-small.png "Adding another backend to the NodeBalancer's configuration")](nodebalancers-backend-nodes2.png)
+
+Once the configuration is sent to the backend, users will be transparently balanced over the two Linodes and each will be monitored for health. This configuration is very easy to work with, as upgrades can be rolled out to each backend without disrupting service and backends can be taken in and out of rotation at will.
```

Comment:
Transparently implies that we tell them how this is being done, and I don't think that we do.